packages/firebase_remote_config/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -63,17 +52,17 @@ class RemoteConfig extends FirebasePluginPlatform with ChangeNotifier {
   /// If no successful fetch has been made a [DateTime] representing
   /// the epoch (1970-01-01 UTC) is returned.
   DateTime get lastFetchTime {
-    return _delegate.lastFetchTime;
+    return _delegate.lastFetchTime!;
```

Comment:
The delegate should return a non-nullable value instead (same for the other similar cases)

Reply:
fix